frontend/src/Pages/Home/InputForm/InputForm.tsx

```diff
@@ -167,22 +211,49 @@ const InputForm = () => {
           error={formik.touched.city && Boolean(formik.errors.city)}
           helperText={formik.touched.city && formik.errors.city}
         />
-        <TextField
-          id="relevantSkills"
+        <FormControl
           className="input-form"
-          name="relevantSkills"
-          label="Relevant Skills"
-          value={formik.values.relevantSkills}
-          onChange={formik.handleChange}
-          onBlur={formik.handleBlur}
           error={
             formik.touched.relevantSkills &&
             Boolean(formik.errors.relevantSkills)
           }
-          helperText={
-            formik.touched.relevantSkills && formik.errors.relevantSkills
-          }
-        />
+        >
+          <Autocomplete
+            multiple
+            id="tags-outlined"
+            options={skills}
+            getOptionLabel={(option) => option}
+            value={formik.values.relevantSkills}
+            onChange={handleRelevantSkillsChange}
+            onFocus={() => setAutoCompleteFocus(true)}
```

Comment:
I'm not sure onFocus or onBlur here is needed. Formik should be able to handle the onfocus and onblur naturally. I tried deleting lines 228 and 229 manually and it worked fine for me, pls lmk otherwise

Reply:
same, removed this code and it works.